include/seqan3/io/alignment_file/format_sam.hpp

```diff
@@ -349,6 +367,12 @@ class alignment_file_format_sam
         auto const tab_or_end = is_char<'\t'> || is_char<'\r'> || is_char<'\n'>;
         read_field(stream_view | view::take_until_or_throw(tab_or_end), qual);
 
+        if constexpr (!detail::decays_to_ignore_v<seq_type> && !detail::decays_to_ignore_v<qual_type>)
+            if (std::ranges::distance(seq) != 0 && std::ranges::distance(qual) != 0 &&
+                std::ranges::distance(seq) != std::ranges::distance(qual))
+                throw format_error{to_string("Sequence length (", std::ranges::distance(seq), ") and quality length (",
+                                             std::ranges::distance(qual), ") must be the same.")};
```

Comment:
{}

Reply:
where? format_error has {} and `to_string` is a function

Reply:
I meant around the block :laughing: 
Because multi-line...